examples/create-react-app/src/pages/Admin/media-queries.css

```diff
@@ -0,0 +1,67 @@
+@media (max-width: 800px) {
+    .rainbow-admin_vertical-navigation {
+        width: 200px;
+    }
+
+    .rainbow-admin_main-inner {
+        margin: 75px 0 0 200px;
+        padding: 1.5rem;
+    }
+}
+
+@media (max-width: 700px) {
```

Comment:
Why you use ```@media```  for ```max-width: 800px```,  ```max-width: 700px``` and ```max-width: 600px```?
You should better define the breakpoints. Just use a breakpoint for large screens, one for a medium screen and another for a small screen.